### ISSUE TITLE
Bug: {{ focus_css }} returns empty when no focal point is set

### ISSUE THREAD

**Issue by FrittenKeeZ** [statamic/v2-hub#1577]
### Expected behaviour
When using  {{ focus_css }} I except it to return '50% 50%', when I haven't set it manually.

### Actual behaviour
The tags returns an empty string, resulting in a broken css style "background-position: ;"

### Steps to reproduce
1. Use {{ focus_css }} on an image with no focal point set.
2. See that the style tag is broken
3. Install the FocalPoint addon
4. Use {{ focal_point }} instead
5. ???
6. Profit

### Server Details

**Statamic Version:**
2.6.8

**Updated from an older Statamic or fresh install:**
2.6.6

**List of installed addons:**
FocalPoint

### Bonus
Replace line 104 in `Statamic\View\BaseModifiers` from `return $value;` to `return '50% 50%';`.

**Comment by jackmcdade** [comment]
Good idea. In the mean time you should be able to do `{{ focus_css or "50% 50%" }}`.

**Comment by jasonvarga** [comment]
If there's no `focus` value, then it's blank. It's always been that way. We can change it though.
